api/src/schemas.py

```diff
@@ -167,16 +173,11 @@ class UpdateIngestionRequest(BaseModel):
     message: str = None
 
 
-class Discovery(str, enum.Enum):
-    s3 = "s3"
-    cmr = "cmr"
-
-
 class WorkflowInputBase(BaseModel):
-    collection: str
-    discovery: Discovery
+    collection: str = ""
```

Comment:
does this not essentially make the collection param optional? is that desired behavior?

Reply:
Technically yes, but in practice we have other checks in place to determine whether this field is valid. I made this change so that a `Dataset` could be defined without necessarily including the collection id with each item.